components/index/CardsMonitoring/MonitoringItemsOverview/Card.vue

```diff
@@ -124,6 +132,10 @@ export default {
   @include button-text('sm');
 }
 
+.buttonWrap {
```

Comment:
```suggestion
.button-wrap {
```

こちらもkebab-caseにしておいてもらえると助かります。